app/forms.py

```diff
@@ -129,9 +129,7 @@ class SharedMessageForm(Form):
     Form for submittin a message to share with the network.
     '''
 
-    message = TextAreaField(
-        label=lazy_gettext('Have a question or something to share about innovation?')
```

Comment:
We're custom-rendering this form in `__base_ui__.html` now, so don't worry about labels in the form...